docker/README.md

```diff
@@ -62,7 +62,7 @@ docker run --gpus all --rm -it -v `pwd`:`pwd` -w `pwd` -u $(id -u):$(id -g) devi
 
 #### AMD
 
-Unlike NVidia, AMD does not require an additional docker setup and runs with the standard docker. You will however need to pass some flags so that the image is linked to the GPU devices. You can find a short walkthrough in these [AMD notes](https://developer.amd.com/wp-content/resources/ROCm%20Learning%20Centre/chapter5/Chapter5.3_%20KerasMultiGPU_ROCm.pdf) for their tensorflow GPU docker image.
+Unlike NVidia, AMD does not require an additional docker setup and runs with the standard docker. You will, however, need to pass some flags so that the image is linked to the GPU devices. You can find a short walkthrough in these [AMD notes](https://developer.amd.com/wp-content/resources/ROCm%20Learning%20Centre/chapter5/Chapter5.3_%20KerasMultiGPU_ROCm.pdf) for their TensorFlow GPU docker image.
```

Comment:
docker or Docker ?

spelling should be consistent across the README

perhaps we want the capital letter

what do they use on their website? I suppose Docker (and no need for quotes `)

Reply:
Sure